packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { KeyboardEventHandler, useEffect, useState } from 'react'
+import {
+  Box,
+  List,
+  Paper,
+  Typography,
+  FormControl,
+  InputAdornment,
+  IconButton,
+} from '@mui/material'
+import { useTheme } from '@mui/material'
+import { ColorPartial } from '@mui/material/styles/createPalette'
+import { OutlinedInput } from '@mui/material'
+import { ArrowUpward } from '@mui/icons-material'
+import SynapseChatInteraction from './SynapseChatInteraction'
+import { SkeletonParagraph } from '../Skeleton'
+import { useCreateAgentSession } from 'src/synapse-queries/chat/useChat'
```

Comment:
surprised this non-relative import works

Reply:
me too!  I'll fix it (I'm sure it will break at some point)